Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/UserController.cs

```diff
@@ -479,30 +587,26 @@ public IActionResult UserDetailById(string userId)
                     foreach (var error in apiEx.Errors)
                         _logger.LogInformation("Forgot Password mail not send " + error);
                 }
-                throw apiEx;
+                return BadRequest();
             }
             catch (ArgumentNullException argEx)
```

Comment:
If SendGridApi Key and From values are not set in appsettings then we get an ArgumentNullException.